web/config/sync/image.style.large.yml

```diff
@@ -5,13 +5,13 @@ dependencies: {  }
 _core:
   default_config_hash: J2n0RpFzS0-bgSyxjs6rSdgxB1rb-bTAgqywNx_964M
 name: large
-label: 'Large (480×480)'
```

Comment:
These sizes may be good for now but we should talk to the rest of the team whether they fit their use cases.